.appveyor.yml

```diff
@@ -18,16 +18,9 @@ environment:
       PACKAGES: "-P libgmp-devel,zlib-devel"
       CFLAGS: "--coverage"
       LDFLAGS: "--coverage"
-# FIXME: HPC-GAP builds on AppVeyor are disabled for now, as they experience
-# weird errors in building ward (which I cannot replicate on a Windows VM).
-#    - CYG_ARCH: x86
-#      CYG_ROOT: C:\cygwin
-#      ABI: 32
-#      HPCGAP: yes
-#      BUILDDIR: build       # HPC-GAP only supports kernel extensions for of out-of-tree builds
-#    - CYG_ARCH: x86_64
-#      CYG_ROOT: C:\cygwin64
-#      HPCGAP: yes
+   - CYG_ARCH: x86_64
+     CYG_ROOT: C:\cygwin64
+     HPCGAP: yes
```

Comment:
I am not sure enabling HPC-GAP builds for Windows is worth it at this stage, but I wanted to try what happens; I might remove this again in the final version of this PR.